lefthook.yaml

```diff
@@ -3,5 +3,3 @@ pre-commit:
   commands:
     eslint:
       run: yarn lint-fix
-    format:
-      run: yarn format-fix
```

Comment:
I removed `format` & `format-fix` script commands from `package.json` and lefthook. The reasons are:

1. They did not work properly as I tested them (`yarn format-fix` had no effect on `.vue` files when I added random new lines to the random files).
2. `yarn lint-fix` does the job of formatting since eslint is extended by prettier plugin in `.eslintrc.js` file.
3. Lefthook log in terminal was not readable with the `format` command run, eslint alone returns a short and much more informative log.